src/Tasks/Resources/Strings.resx

```diff
@@ -1338,6 +1338,10 @@
   <data name="RemoveDir.SkippingNonexistentDirectory">
     <value>Directory "{0}" doesn't exist. Skipping.</value>
   </data>
+  <data name="RemoveDir.EmptyPath">
+    <value>MSB3232: Detected an empty directory. Skipping.</value>
```

Comment:
Empty directory can also mean that the current directory is empty. It should mean that the path value is empty.

```suggestion
    <value>MSB3232: Detected an empty directory path. Skipping…</value>
```

**OR**

```suggestion
    <value>MSB3232: Directory path should not be empty. Skipping empty path…</value>
```

Wording could be better... But it's a start!